src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
```

Comment:
I forgot to say in our chat - I spoke to Rosie about the best title for the page and we think ‘Resources and tools’ might be good for super-clarity.